queries.sql

```diff
@@ -24,4 +24,70 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 /*Find all animals with a weight between 10.4kg and 17.3kg 
 (including the animals with the weights that equals precisely
  10.4kg or 17.3kg)*/
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made.*/
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
+
+/*Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.*/
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+COMMIT;
+
+/*Update the animals table by setting the species column to pokemon for all animals that don't have species already set.*/
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+/*Verify that changes were made.*/
+SELECT * FROM animals;
+
+/*Commit the transaction*/
+COMMIT;
+
+/*Verify that changes persist after commit.*/
+SELECT * FROM animals;
+
+/*Delete animals */
+DELETE FROM animals;
+ROLLBACK;
+
+/*Delete all animals born after Jan 1st, 2022.*/
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+-- Create a savepoint for the transaction.
+SAVEPOINT SP;
+
+--Update all animals' weight to be their weight multiplied by -1.
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] Please make sure each transaction starts with `BEGIN` as you did for the first transaction.